src/components/ui/DatePicker/DatePicker.svelte

```diff
@@ -268,13 +268,13 @@
         <div>
           <div class="action button" on:mousedown={setToday}>
             <div class="action-icon"><Calendar /></div>
-            <div class="action-label">Today</div>
+            <div class="action-label st-typography-body">Today</div>
           </div>
         </div>
         <div>
           <div class="action button" on:mousedown={clearDate}>
-            <div class="action-icon"><i class="si si-magic_eraser" /></div>
-            <div class="action-label">Clear</div>
+            <div class="action-icon"><i class="bi bi-magic" /></div>
```

Comment:
please keep `si si-magic_eraser` as that's the new icon to use.

Reply:
Was so sure I was taking your changes.. think there was some vscode update to the merge view that definitely threw me off.